File: common-core/src/main/java/org/example/commoncore/dto/OrderItemsDetailDto.java
package org.example.commoncore.dto;

import lombok.*;

import java.util.UUID;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class OrderItemsDetailDto {

    private UUID productId;

    private UUID warehouseId;

    private Integer quantity;
}

File: order-service/src/main/kotlin/org/example/orderservice/dto/OrderItemsDetailDto.kt
package org.example.orderservice.dto

import java.util.*

data class OrderItemsDetailDto(
    val productId: UUID?,
    val warehouseId: UUID?,
    val quantity: Int?,
) {

    constructor() : this(
        productId = null,
        warehouseId = null,
        quantity = null,
    )
}
